e2e/wdio/driver/wdio.ts

```diff
@@ -57,28 +66,36 @@ export class Wdio {
     };
 
     waitForInvisibilityOf(selector: string, index: number = 0): boolean {
-        return $$(selector)[index].waitForDisplayed({ reverse: true })
+        return $$(selector)[index].waitForDisplayed({ reverse: true });
     }
 
     waitForNotDisplayed(selector: string, waitTime = this.defaultWaitTime): boolean {
-        return $(selector).waitForDisplayed({timeout: waitTime, reverse: true});
+        return $(selector).waitForDisplayed({ timeout: waitTime, reverse: true });
     };
 
+    waitForNotFocused(selector: string, index: number = 0, waitTime = this.defaultWaitTime):void {
```

Comment:
Maybe use waitUntil with isFocused check inside. This one isn't waiting as for me. If it is a generic check let's put it in the assertion helper. 
expect is not belongs to webDriver class.